src/scripts/extensions/authenticationHelper.ts

```diff
@@ -136,12 +136,18 @@ export class AuthenticationHelper {
 	/**
 	 * Makes a call to the authentication proxy to retrieve the user's information.
 	 */
-	public static retrieveUserInformation(clipperId: string, cookie: string = undefined): Promise<ResponsePackage<string>> {
+	public static retrieveUserInformation(clipperId: string, cookie: string = undefined, logger: Logger = undefined): Promise<ResponsePackage<string>> {
 		return new Promise<ResponsePackage<string>>((resolve, reject: (error: OneNoteApi.RequestError) => void) => {
 			let userInfoUrl = Utils.addUrlQueryValue(Constants.Urls.Authentication.userInformationUrl, Constants.Urls.QueryParams.clipperId, clipperId);
 
+			let retrieveUserInformationEvent = new Log.Event.PromiseEvent(Log.Event.Label.RetrieveUserInformation);
```

Comment:
Since you most actively look at this data, you would know best: might it be easier to debug auth issues if you added this new info to the existing `GetExistingUserInformation` event? Or is it helpful to keep them separate? There just seems to be large overlap between when these two events will be logged.